contrib/scripts/orphan-clean-up/README.md

```diff
@@ -0,0 +1,19 @@
+# Orphan Clean Up
+
+## Overview
+
+The Orphan Clean Up script is a basic PowerShell script to delete orphaned entities in the catalog.
```

Comment:
This will need a disclaimer saying something like;

Warning; There is a risk of entities being orphaned(and being deleted by this script) in case of the location having problems and returning a 404 status code. This might lead to accidental deletion of entities until the processing loop has recreated the entity.

Reply:
Sounds good, I've added this